example.env

```
@@ -1 +1,2 @@
-DISCORD_TOKEN=BLAHBLAH
\ No newline at end of file
+DISCORD_TOKEN=BLAHBLAH
```

[nit] you can probably remove the BLAHBLAH if we have another example without anything in it

```suggestion
DISCORD_TOKEN=
```